test/verify/check-machines

```diff
@@ -370,6 +375,68 @@ class TestMachines(MachineCase):
             print("Libvirt version does not support disk statistics")
             b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
 
+    def testLibvirt(self):
+        b = self.browser
+        m = self.machine
+
+        libvirtServiceName = self.getLibvirtName()
+
+        def checkLibvirtEnabled():
+            try:
+                m.execute("systemctl -q is-enabled {0}".format(libvirtServiceName))
+                return True
+            except subprocess.CalledProcessError:  # return code != 0
+                return False
+
+        self.startVm("subVmTest1")
+        self.login_and_go("/machines")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        m.execute("systemctl disable {0}".format(libvirtServiceName))
+        m.execute("systemctl stop {0}".format(libvirtServiceName))
+
+        b.wait_present("#slate-header")
+        b.wait_in_text("#slate-header", "Virtualization Service (libvirt) is Not Active")
+        b.wait_present("#enable-libvirt:not(:checked)")
+        b.set_checked("#enable-libvirt", True)
+        b.click("#start-libvirt")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait(lambda: checkLibvirtEnabled())
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        m.execute("systemctl stop {0}".format(libvirtServiceName))
+        b.wait_present("#slate-header")
+        b.wait_in_text("#slate-header", "Virtualization Service (libvirt) is Not Active")
+        b.wait_present("#enable-libvirt:checked")
+        b.set_checked("#enable-libvirt", False)
+        b.click("#start-libvirt")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait(lambda: not checkLibvirtEnabled())
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        m.execute("systemctl enable {0}".format(libvirtServiceName))
+        m.execute("systemctl stop {0}".format(libvirtServiceName))
+
+        b.wait_present("#slate-header")
+        b.wait_in_text("#slate-header", "Virtualization Service (libvirt) is Not Active")
+        b.wait_present("#enable-libvirt:checked")
+
+        b.click("#troubleshoot")
+        b.leave_page()
+        url_location = "/system/services#/{0}".format(libvirtServiceName)
+        b.wait(lambda: url_location in b.eval_js("window.location.href"))
+
+        # HACK: restarting libvirtd causes this completely unrelated SELinux issue
+        self.allow_journal_messages('.*denied.*search.*"systemd-machine".*dev="proc".*')
+
+        self.allow_journal_messages('.*audit.*apparmor="DENIED".*operation="signal".*libvirt.*denied_mask="send".*')
```

Comment:
These two still need to be replaced with known issues.

Reply:
should I create these issues and put these messages into naughty overrides in this PR?

Reply:
Yes, please.

Reply:
maybe we could reopen this issue https://github.com/cockpit-project/cockpit/issues/8467 ? It is exactly the same except the signal is `hup` instead of  `kill`

Reply:
I'm quite sure the policy is not so  specific as to limit particular signals, it's just not allowed to send signals in general. I. e. it's most likely the same bug. But are you sure that it still actually applies? This was fixed recently (certainly way after you started working on this PR), and in the recent  [test run](https://fedorapeople.org/groups/cockpit/logs/pull-8493-20180322-185454-2e89d3c5-verify-debian-testing/log.html#158) there is no such apparmor denial. So I think this can just be dropped and should be fine as-is.

Reply:
I still see it in the logs, but it is fine with me if it is not causing any problems anymore.